### PR TITLE
Calib

### PR DESCRIPTION
**Pull request by fbenke-pik**
First version of a reporting script comparing a baseline run to its calibration

### CODE REVIEW

**Review by chroetz — CHANGES_REQUESTED** [review]

File: R/checkVsCalibData.R [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#' # Simple use. Creates PDF:
+#' checkVsCalibData(
+#'   gdx = "path/to/data.gdx",
+#'   outputDir = "path/to/output/directory",
+#'   outputFile = "Check_vs_CalibData_Example.pdf"
+#' )
+#' }
+#' @export
+checkVsCalibData <- function(gdx, outputDir = getwd(), outputFile = "Check_vs_CalibData.pdf") {
+  yamlParams <- c(list(gdx = normalizePath(gdx, mustWork = TRUE)))
```
[the remark this answers is not in the document]
Comment (fbenke-pik) [reply R/checkVsCalibData.R:21]: no, they were just a leftover, i dont need the `...` argument. hence, removing the `c()`

File: inst/markdown/checkVsCalibData/rc_main.Rmd [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+library(tidyverse)
+library(kableExtra)
+library(quitte)
+library(gdx)
+library(ggplot2)
+library(dplyr)
+
+
+min.reldev <- 15
+min.absdev <- 1e-1
```
[the remark this answers is not in the document]
Comment (fbenke-pik) [reply inst/markdown/checkVsCalibData/rc_main.Rmd:51]: for now, we don't need to make them adjustable. they are constants.

File: inst/markdown/checkVsCalibData/rc_main.Rmd [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  figHeight: 10
+  warning: no
+  message: no
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = FALSE,
+  error = TRUE,
+  message = FALSE,
```
[the remark this answers is not in the document]
Comment (fbenke-pik) [reply inst/markdown/checkVsCalibData/rc_main.Rmd]: has been adjusted

File: inst/markdown/checkVsCalibData/rc_main.Rmd [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+The following table lists all deviations greater 15 %.
+
+```{r quantity deviations}
+
+# get deviations
+
+deviations <- left_join(
+  as.quitte(q.baseline) %>% select(region, variable = all_in, period, value) %>% mutate(model := "Baseline"),
+  as.quitte(q.calibration) %>% select(region, variable = all_in, period, value) %>% mutate(model := "Calibration"),
```
[the remark this answers is not in the document]
Comment (fbenke-pik) [reply inst/markdown/checkVsCalibData/rc_main.Rmd]: fixed

**Review by fschreyer — APPROVED** [review]

**Pull request merged**